Specs/Core/CesiumTerrainProviderSpec.js

```diff
@@ -254,8 +253,12 @@ describe("Core/CesiumTerrainProvider", function () {
     const provider = new CesiumTerrainProvider({
       url: "made/up/url",
     });
-    expect(provider.errorEvent).toBeDefined();
-    expect(provider.errorEvent).toBe(provider.errorEvent);
+    return pollToPromise(function () {
+      return provider.ready;
```

Comment:
No need to change this, but I just wanted to mention that polling for `ready` should be equivalent to awaiting the `readyPromise`.